relay_messages.go

```diff
@@ -132,6 +134,10 @@ func newLazyCallReq(f *Frame) lazyCallReq {
 	arg1Len := int(binary.BigEndian.Uint16(f.Payload[cur : cur+2]))
 	cur += 2
 	cr.method = f.Payload[cur : cur+arg1Len]
+
+	// arg2~2
+	cur += arg1Len
+	cr.arg2EndOffset = cur + 2 /*arg2 len*/ + int(binary.BigEndian.Uint16(f.Payload[cur:cur+2]))
```

Comment:
if the frame has a continue bit, then arg2 offset is past this. should we somehow flag this to the caller as well?

Reply:
Good point - added isFragmented in the Arg2EndOffset method to indicate if Arg2 is fragmented.